main.py

```diff
@@ -1,6 +1,43 @@
 """
 Codebreaker
 """
+import cProfile
+def dr_test(x, y, box, code, old_guesses, count):
+    for e in range(x):
+        guess_code = []
+        hint = []
+        print('Pick from R, B, Y, G, O, or P')
+        print('Type quit to end early')
+        for i in range(y):
+            guess = (input(f'Guess the {i + 1} position: ')).upper()
+            if guess == 'QUIT':
+                return print("You ended early, better luck next time")
+            while guess not in set(box):
```

Comment:
getting rid of the repeating != x,y,z 

this set() gets the unique values from a that box